mkt/reviewers/tests/test_views.py

```diff
@@ -4146,3 +4139,114 @@ def test_comm_url_no_encode(self):
+
+    def setUp(self):
+        super(ModerateLogTest, self).setUp()
+        self.review = Review.objects.create(addon=app_factory(), body='body',
+                                            user=user_factory(), rating=4,
+                                            editorreview=True)
+        mkt.set_user(UserProfile.objects.get(username='editor'))
+
+
+class TestModerateLog(ModerateLogTest, AccessMixin):
+
+    def setUp(self):
+        super(TestModerateLog, self).setUp()
+        self.url = reverse('reviewers.apps.moderatelog')
+
+    def test_log(self):
+        r = self.client.get(self.url)
+        eq_(r.status_code, 200)
+
+    def test_start_filter(self):
+        r = self.client.get(self.url, dict(start='2011-01-01'))
+        eq_(r.status_code, 200)
+
+    def test_enddate_filter(self):
+        """
+        Make sure that if our end date is 1/1/2011, that we include items from
+        1/1/2011.
+        """
+        mkt.log(mkt.LOG.APPROVE_REVIEW, self.review, self.review.addon,
+                created=datetime(2011, 1, 1))
+
+        r = self.client.get(self.url, dict(end='2011-01-01'))
+        eq_(r.status_code, 200)
+        eq_(pq(r.content)('tbody td').eq(0).text(), 'Jan 1, 2011 12:00:00 AM')
+
+    def test_action_filter(self):
+        """
+        Based on setup we should see only two items if we filter for deleted
+        reviews.
+        """
+        for i in xrange(2):
+            mkt.log(mkt.LOG.APPROVE_REVIEW, self.review.addon, self.review)
+            mkt.log(mkt.LOG.DELETE_REVIEW, self.review.addon, self.review)
+        r = self.client.get(self.url, dict(search='deleted'))
+        eq_(pq(r.content)('tbody tr').length, 2)
+
+    def test_no_results(self):
+        r = self.client.get(self.url, dict(end='2004-01-01'))
+        no_results = 'No events found for this period.'
+        assert no_results in r.content, 'Expected no results to be found.'
+
+    def test_display_name_xss(self):
+        mkt.log(mkt.LOG.APPROVE_REVIEW, self.review, self.review.addon,
+                user=self.admin_user)
+        self.admin_user.display_name = '<script>alert("xss")</script>'
+        self.admin_user.save()
+        assert '<script>' in self.admin_user.display_name, (
+            'Expected <script> to be in display name')
+        r = self.client.get(self.url)
+        inner_html = pq(r.content)('#log-listing tbody td').eq(1).html()
+        assert '<script>' not in r.content
+        assert '&lt;script&gt;' in r.content
+
+
+class TestModerateLogDetail(ModerateLogTest, AccessMixin):
+
+    def setUp(self):
+        super(TestModerateLogDetail, self).setUp()
+        # AccessMixin needs a url property.
+        self.url = self._url(0)
+
+    def _url(self, id):
+        return reverse('reviewers.apps.moderatelog.detail', args=[id])
+
+    def test_detail_page(self):
+        mkt.log(mkt.LOG.APPROVE_REVIEW, self.review.addon, self.review)
+        id = ActivityLog.objects.editor_events()[0].id
```

Comment:
nit: id is a built-in function, so I like to avoid using it as a variable/argument name when possible (also happens below in the view, and in other tests)